sharktank/sharktank/utils/testing.py

```diff
@@ -21,6 +22,8 @@
 from ..types import *
 from .math import cosine_similarity
 
+is_mi300x = pytest.mark.skipif("config.getoption('iree_hip_target') != 'gfx942'")
```

Comment:
Moving this into the utils file is nice. Could pull that out into its own PR if you want it merged sooner.

Could add similar marks for other cases too.